dosomething-qa/fastly-frontend/projectpages_error.vcl

```diff
@@ -0,0 +1,6 @@
+if (obj.status == 779) {
+  # Redirect any requests to */project/* and send them to the homepage.
+  set obj.http.Location = "/us"
```

Comment:
Oops, another sneaky syntax error here:

```
* module.dosomething-qa.module.fastly-frontend.fastly_service_v1.frontend-qa: 1 error(s) occurred:

* fastly_service_v1.frontend-qa: [ERR] Invalid configuration for Fastly Service (5I35is1gX0tbds1VfNmBIb): Syntax error: Expected variable, string or semicolon got `set`
at: (input Line 1079 Pos 3)
  set obj.status = 302;
--###------------------
```

We need a semi-colon on the line above here (after `"/us"`).